instance/models/utils.py

```diff
@@ -182,7 +182,7 @@ def __eq__(self, obj):
         To compare with inherited classes etc., use a full isinstance or issubclass expression.
         """
         if inspect.isclass(obj):
-            return type(self) is obj  # pylint: disable=unidiomatic-typecheck
+            return type(self) is obj
```

Comment:
This isn't idiomatic because of Python's idiosyncrasies :slightly_smiling_face:

Reply:
That's a fine tautology, I'll keep this in mind :smile: